src/shared/components/events-list/index.js

```diff
@@ -2,134 +2,123 @@
-                        <Grid fit={false}>
-                          <Cell size={8} key='event_description' breakOn="mobileS">
-                            <a className={styles.eventTitleLink} href={eventHref(event)}>
-                              <h2 className={styles.eventTitle}>
-                                <span>
-                                  {event.title}
-                                </span>
-                                <span className={classNames(
-                                  {
-                                    [styles.arrow]: true,
-                                    [icons.sketchArrowRight]: true,
-                                  })}
-                                />
-                              </h2>
-                            </a>
-                            <div className={styles.eventDescription}>
-                              {event.strapline}
-                            </div>
-                            {
-                              event.externalLinks || event.internalLinks ?
-                                <div className={styles.eventLinks}>
-                                {
-                                  event.externalLinks ?
-                                    <EventLinksList
-                                      linkList={event.externalLinks}
-                                      listType="external" />
-                                    : null
-                                }
-                                {
-                                  event.internalLinks ?
-                                    <EventLinksList
-                                      linkList={event.internalLinks}
-                                      listType="internal" />
-                                    : null
-                                }
-                                </div>
-                              : null
-                            }
-                            {
-                              event.tags
-                              ? <TagsList
-                                  tags={event.tags}
-                                  tagsLinkPath="about-us/events" />
-                              : null
-                            }
-                          </Cell>
-                          <Cell size={4} key='event_picture' breakOn="mobileS" hideOn="mobileS">
-                            <EventImage imgPath={ imageAssetsEndpoint + (event.featureImageFilename ? event.featureImageFilename : 'red-badger-event.jpg') } href={eventHref(event)} />
-                          </Cell>
-                        </Grid>
-                      </Cell>
-                    </Grid>
-                  </li>
-              ))
-            }
-          </ul>
-        </div>
-      );
-    }
+EventsList.propTypes = {
+  events: PropTypes.arrayOf(React.PropTypes.object).isRequired,
+  timeline: PropTypes.oneOf(['past', 'future', 'today']),
```

Comment:
Also take the props from the child rather than duplicating the definition.

```javascript
timeline: EventsTimeline.propTypes.timeline.isRequired,
```